### DISCUSSION TITLE
DefaultHelpers url_for

### DISCUSSION THREAD

**Discussion by s1037989** [mojolicious/mojo#1720] [Q&A] [answered]
Does [the example with controller and action in DefaultHelpers url_for](https://docs.mojolicious.org/Mojolicious/Plugin/DefaultHelpers#url_for) still apply to Mojolicious 9.0?

**Comment by jberger** [comment, marked as answer]
I suspect that should be changed, yes

**Reply by kraih** [reply]
That needs to be changed indeed.
